.github/workflows/codesee-arch-diagram.yml

```diff
@@ -4,14 +4,12 @@ on:
       - main
   pull_request:
     types: [opened, synchronize, reopened]
-    branches-ignore:
-      - 'renovate/**'
-      - 'i18n-sync-**'
 
 name: CodeSee Map
 
 jobs:
   test_map_action:
+    if: ${{ github.actor != 'renovate[bot]' && github.actor != 'camperbot' }}
```

Comment:
Neat.  How did you know/figure out that it was `renovate[bot]` not just `renovate`?

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/blob/b5019000bbfe12dfccfc144293d69285e237d756/.github/workflows/codeql-analysis.yml#L13

Example 🤷‍♂️

Reply:
Oh.  Well.  Fine. :man_facepalming:

Reply:
Checked my own custom github tooling lol 

(To get it initially for that I queried the API and looked at the value)

Reply:
> queried the API 

So, what, make a query like [this](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request) and just look at the login?